docs/support_files/world.md

```diff
@@ -33,4 +33,11 @@ defineSupportCode(function({setWorldConstructor}) {
 });
 ```
 
+If you want to be able to attach screenshots to the results file you will need to pass in {attach} explicity to your CustomWorld.
+
+```javascript
+function CustomWorld({attach}) {...}
+  this.attach = attach;
+```
```

Comment:
Can we just document what exactly the arguments that are passed to the world constructor are in the default world constructor above?